package.json

```diff
@@ -26,18 +26,21 @@
   "homepage": "https://github.com/omise/omise-node",
   "dependencies": {
     "@types/node": "^8.10.66",
+    "dotenv": "^16.4.5",
```

Comment:
Is this suppose to be here or under dev dependencies?

Reply:
Should be inside dev, i will update it.